packages/snaps-utils/src/handlers.ts

```diff
@@ -132,19 +140,23 @@ export const OnTransactionResponseStruct = nullable(
 
 export const OnSignatureResponseStruct = OnTransactionResponseStruct;
 
-export const OnHomePageResponseWithContentStruct = object({
+export const OnPageResponseWithContentStruct = object({
```

Comment:
Since these are exported, it's perhaps simpler to not rename them. Not sure why they are exported in the first place

Reply:
What's the problem with it ? I just created an "universal" one and bound it to the ones we had before down in the file :thinking:

Reply:
It just lets us avoid a breaking change if we don't rename them